### PR TITLE
lwimiq

### PR DESCRIPTION
**Pull request by durandom**
et voila

### CODE REVIEW

**Comment by durandom** [comment]
@cybette please review. And as I'm out sick today feel free to adjust wording yourself :)

File: site/_posts/2017-06-12-Where-have-all-the-providers-gone.md [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+
+Some metrics seemed to be useless, so we dont calculate them anymore, [click](https://github.com/ManageIQ/manageiq/pull/15260)
+
+## Wrapping up
+
+That's it. Really. Move over, nothing more to see. :wave: Yep, we're out. Roger, Over and, what was his name? Out. Yes. Word!
+
+See you next week. 
+
+[manageiq PRs merged]: https://github.com/ManageIQ/manageiq/pulls?page=1&q=is%3Apr+is%3Amerged+base%3Amaster+merged%3A%222017-06-05+..+2017-06-12%22+sort%3Acreated-desc&utf8=%E2%9C%93
```
Comment (cybette) [inline site/_posts/2017-06-12-Where-have-all-the-providers-gone.md]: This and the following 16 link definitions are not used in the post. I don't see the corresponding link labels anywhere. Did you mean to put them in a list somewhere in the post?

**Comment by cybette** [comment]
@durandom Great post! I enjoyed reading it :) Just came back from a music festival over the weekend so it was rather fitting as well :metal: :smile: 

@jntullo @imtayadeway @syncrou would you take a look-see as well? Thanks!

**Comment by jntullo** [comment]
Awesome post! 🎶  LGTM!

File: site/_posts/2017-06-12-Where-have-all-the-providers-gone.md (hunk 1 above)
Comment (imtayadeway) [inline site/_posts/2017-06-12-Where-have-all-the-providers-gone.md]: They are auto-generated by the generator :)

File: site/_posts/2017-06-12-Where-have-all-the-providers-gone.md [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+author: durandom
+date: 2017-06-12
+comments: true
+published: true
+tags: LWIMIQ
+---
+:guitar: Before you continue reading, I want you to click [here](https://youtu.be/ZgXNVA9ngx8) to set the mood :metal:
+
+```
+Where have all the Providers gone?
```
Comment (imtayadeway) [inline site/_posts/2017-06-12-Where-have-all-the-providers-gone.md]: :heart: I don't want to admit how often I parody this song in my head :laughing:

File: site/_posts/2017-06-12-Where-have-all-the-providers-gone.md (hunk 1 above)
Comment (cybette) [inline site/_posts/2017-06-12-Where-have-all-the-providers-gone.md]: @imtayadeway yeah I later noticed :P so it's more of a convenience for the authors and they can use the link labels if they wish?

**Comment by cybette** [comment]
Ok, looks good to merge. Thanks for the reviews! @durandom feel better soon :)

File: site/_posts/2017-06-12-Where-have-all-the-providers-gone.md (hunk 1 above)
Comment (imtayadeway) [inline site/_posts/2017-06-12-Where-have-all-the-providers-gone.md]: @cybette I think the way it started out, I'd link to the main repo in the preamble with some text like "we had x number of PRs merged this week from y contributors" etc... I think that doing that now might be a little dreary as there are so many new repos....we could bake a table into the generator but then that might look a lot like the spring review. But yes....they still serve a purpose of at the very least providing links for the author to go examine what changed and can be deleted if not needed, though they do no harm if you leave them in. Any thoughts?

**Pull request merged**